src/main/java/Event.java

```diff
@@ -0,0 +1,20 @@
+public class Event extends Task {
+    private String timing;
+
+    public String getTiming(){
+        return this.timing;
+    }
+
+    public Event(String description, String timing){
+        super(description);
+        this.timing = timing;
+    }
+
+    public void setTiming(String timing) {
+        this.timing = timing;
+    }
+    @Override
+    public void printItem() {
+        System.out.println("[E]["+this.getStatusIcon()+"] "+this.description+" (at: "+this.timing+")");
+    }
+}
```

Comment:
Clear and correct naming of all classes, variables and functions as per coding standards.